bin/start_lke.sh

```diff
@@ -0,0 +1,348 @@
+else
+  echo "Create a password for the sirius accounts database"
+  pulumi config set --secret sirius:accounts_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius
+fi
+
+# Sirius Ledger Database
+if pulumi config get sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius >/dev/null 2>&1; then
+  echo "Password found for sirius ledger database"
+else
+  echo "Create a password for the sirius ledger database"
+  pulumi config set --secret sirius:ledger_pwd -C ${script_dir}/../pulumi/python/kubernetes/applications/sirius
+fi
+
+# Admin password for grafana (see note in __main__.py in prometheus project as to why not encrypted)
+# This is for the deployment that is setup as part of the the prometheus operator driven prometheus-kube-stack.
+#
+if pulumi config get prometheus:adminpass -C ${script_dir}/../pulumi/python/config >/dev/null 2>&1; then
+  echo "Password found for grafana admin account"
+else
+  echo "Create a password for the grafana admin user"
+  pulumi config set prometheus:adminpass -C ${script_dir}/../pulumi/python/config
+fi
+
+# TODO: Figure out better way to handle hostname / ip address for exposing our IC #82
+#
+# This version of the code forces you to add a hostname which is used to generate the cert when the application is
+# deployed, and will output the IP address and the hostname that will need to be set in order to use the self-signed
+# cert and to access the application.
+#
+
+echo " "
+echo "NOTICE! Currently we do not automatically pull the hostname of the K8 LoadBalancer with this deployment; instead"
+echo "you will need to create a FQDN and map the assigned IP address to your FQDN in order to use the deployment. "
+echo "You can then add this mapping to DNS, or locally to your host file"
+echo " "
+echo "This will be streamlined in a future release of MARA."
+echo " "
+
+# So we can see...
+sleep 5
+
+if pulumi config get kic-helm:fqdn -C ${script_dir}/../pulumi/python/config >/dev/null 2>&1; then
+  echo "Hostname found for deployment"
+else
+  echo "Create a fqdn for your deployment"
+  pulumi config set kic-helm:fqdn -C ${script_dir}/../pulumi/python/config
+fi
+
+# Show colorful fun headers if the right utils are installed and NO_COLOR is not set
+#
+function header() {
+  if [ -z ${NO_COLOR+x} ]; then
+    "${script_dir}"/../pulumi/python/venv/bin/fart --no_copy -f standard "$1" | "${script_dir}"/../pulumi/python/venv/bin/lolcat
+  else
+    "${script_dir}"/../pulumi/python/venv/bin/fart --no_copy -f standard "$1"
+  fi
+}
+
+#
+# The initial version of this tried to manage the kubernetes configuration file, but for some reason
+# Linode is a bit touchy about this.
+#
+# So, now we just backup the existing file and slide ours in place. This will be streamlined/addressed as
+# part of the rewrite...
+#
+function add_kube_config() {
+  echo "adding ${cluster_name} cluster to local kubeconfig"
+  mv ~/.kube/config ~/.kube/config.mara.backup || true
```

Comment:
Do not assume that `~` will work properly. Use `$HOME` instead and handle the case where `KUBECONFIG` is loaded.

Reply:
Fixed ~ - going to leave $KUBECONFIG case as-is (since that is consistent with the other start scripts) so we can fix in refactor.